### PR TITLE
Switched to new version of specs

### PR DESCRIPTION
**Pull request by philsturgeon**
With the specs finally getting their own rendering on [openapis.org](https://spec.openapis.org/oas/v3.0.0.html) (instead of living only on swagger.io), lets link to them!

### CODE REVIEW

File: docs/_includes/rules-list.html [hunk 1]
```diff
@@ -14,10 +14,8 @@
     <td>
         {% if rule.object == "*" %}
         <em>everything</em>
-        {% elsif rule.object == "openapi" %}
-        <a href="https://swagger.io/specification/#oasObject">{{ rule.object }}</a>
         {% else %}
-        <a href="https://swagger.io/specification/#{{ rule.object }}Object">{{ rule.object }}</a>
+        <a href="https://spec.openapis.org/oas/v3.0.0.html#{{ rule.object }}-object">{{ rule.object }}</a>
```
Comment (MikeRalphson) [inline docs/_includes/rules-list.html]: There is a `v3.0.1.html` file there if you want to link to that instead.

**Comment by philsturgeon** [comment]
Done!

File: docs/_includes/rules-list.html (hunk 1 above)
Comment (philsturgeon) [reply docs/_includes/rules-list.html]: Updated! Thumb? :D

**Comment by philsturgeon** [comment]
Merging as part of v0.8.2 because everyone refuses to review this one for some reason. 🤣

**Pull request closed without merging**